src/utils/VerificationValidationParser.cpp

```diff
@@ -153,9 +156,62 @@ void Parser::searchFinalDefense(Result* r) {
     }
 }
 
-Result* Parser::lc(const QString* terminalOutput) {
-    return nullptr; // TODO: implement
+Result* Parser::lc(const QString* cmd, const QString* terminalOutput) {
+	Result final;
+	final.terminalOutput = *terminalOutput;
+	final.cmd = *cmd;
+	Result* ptr_final = &final;
+
+	/* Convert terminal output and cmd into string for ease of parsing */
+	string s_terminalOutput = terminalOutput->toStdString();
+	string s_cmd = cmd->toStdString();
+
+	/* Check if length of list is zero, aka does it warnings or errors*/
+	if(stoi(s_terminalOutput.substr(s_terminalOutput.find(":") + 1)) == 0) { // Contains no issues
+		final.resultCode = Result::PASSED;
+		return ptr_final;
+	}
+
+
+	string issueDescription;
+	/* Is it an error or warning? */
+	if(s_cmd.find("-d") != string::npos) { // This is a Warning
```

Comment:
instead of checking the command for flags, try this approach:
![image](https://user-images.githubusercontent.com/65752735/197673971-c326c282-dc90-4a66-ac8b-e53e9dce07b0.png)
![image](https://user-images.githubusercontent.com/65752735/197674080-30907c8a-bee7-4a42-ad51-b6ad98396baf.png)

the lc commands you're working on are already in DefaultTests

- also, just because the command we're running has the flags doesn't mean there's an error
- e.g.: "lc -m all" doesn't necessarily have errors, you have to look at the output results to determine whether there are errors